<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tfm</class>
 <widget class="QMainWindow" name="tfm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>tfm</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <widget class="QTreeView" name="fs_tree">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="showDropIndicator" stdset="0">
         <bool>false</bool>
        </property>
        <property name="rootIsDecorated">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListView" name="bookmarks">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="dragDropMode">
         <enum>QAbstractItemView::DropOnly</enum>
        </property>
        <property name="defaultDropAction">
         <enum>Qt::ActionMask</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableView" name="table_view">
      <property name="cursor" stdset="0">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <property name="acceptDrops">
       <bool>true</bool>
      </property>
      <property name="toolTipDuration">
       <number>-3</number>
      </property>
      <property name="locale">
       <locale language="German" country="Germany"/>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="dragEnabled">
       <bool>true</bool>
      </property>
      <property name="dragDropMode">
       <enum>QAbstractItemView::DragDrop</enum>
      </property>
      <property name="defaultDropAction">
       <enum>Qt::CopyAction</enum>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::ExtendedSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="showGrid">
       <bool>false</bool>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>150</number>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="sizeGripEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QToolBar" name="toolbar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_menu"/>
   <addaction name="action_new_dir"/>
   <addaction name="action_new_file"/>
   <addaction name="action_back"/>
   <addaction name="action_forward"/>
   <addaction name="action_up"/>
   <addaction name="action_home"/>
   <addaction name="action_go"/>
  </widget>
  <action name="action_menu">
   <property name="text">
    <string>Menu</string>
   </property>
  </action>
  <action name="action_new_dir">
   <property name="text">
    <string>New Directory</string>
   </property>
  </action>
  <action name="action_new_file">
   <property name="text">
    <string>New File</string>
   </property>
  </action>
  <action name="action_back">
   <property name="text">
    <string>Back</string>
   </property>
  </action>
  <action name="action_up">
   <property name="text">
    <string>Up</string>
   </property>
  </action>
  <action name="action_home">
   <property name="text">
    <string>Home</string>
   </property>
  </action>
  <action name="action_go">
   <property name="text">
    <string>Go</string>
   </property>
   <property name="toolTip">
    <string>Go or Reload</string>
   </property>
  </action>
  <action name="action_forward">
   <property name="text">
    <string>Forward</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
